samples/commands/execute_command/README.md

```diff
@@ -0,0 +1,529 @@
+the customer, while the data plane operations (the actions performed by the sample application) would be issued by
+software running on the IoT device itself.
+
+Using the IoT Commands service and this sample requires an understanding of two closely-related but different service terms:
+* **AWS IoT Command** - metadata describing a task that the user would like one or more devices to run.
+* **AWS IoT Command Execution** - metadata describing the state of a single device's attempt to execute an AWS IoT Command.
+
+In particular, you can define an **AWS IoT Command** and then send it multiple times to the same device. The device will
+try to execute each received **AWS IoT Command Execution**.
+
+Another important feature of AWS IoT command service is using different MQTT topics for different payload formats. This
+means that your device can choose to receive only AWS IoT commands with specified payload format.
+AWS IoT Commands service distinguishes the following payload formats:
+- JSON
+- CBOR
+- generic (i.e. everything else)
+
+This also means that if your device wants to receive JSON **and** CBOR payloads, it needs to subscribe to both topics
+using two separate API calls.
+
+> [!NOTE]
+> The word `command` is quite overloaded in this sample. It's supposed to be used with multiple meanings:
+> - AWS IoT command - an event with instructions sent from the IoT Core to a device
+> - AWS CLI command - a shell command for executing a control plane operation
+> - sample command - an action that this sample application can perform, such as `open-thing-stream`
+> - there is also AWS IoT command execution.
+>
+> To avoid confusion, the `command` word will allways be used with a context.
+
+### Interaction with sample application
+
+Once connected, the sample supports the following sample commands:
+
+* open-thing-stream <payload-format> - subscribe to a stream of AWS IoT command executions with a specified payload format
+targeting the IoT Thing set on the application startup
+* open-client-stream <payload-format> - subscribe to a stream of AWS IoT command executions with a specified payload format
+targeting the MQTT client ID set on the application startup
+* update-command-execution <execution-id> \<status> \[\<reason-code>] \[\<reason-description>] - update status for specified
+execution ID;
+  * status can be one of the following: IN_PROGRESS, SUCCEEDED, REJECTED, FAILED, TIMED_OUT
+  * reason-code and reason-description may be optionally provided for the REJECTED, FAILED, or TIMED_OUT statuses
+
+Miscellaneous
+* list-streams - list all open streaming operations
+* close-stream <stream-id> - close a specified stream; <stream-id> is internal ID that can be found with 'list-streams'
+* quit - quit the sample application
+
+### Prerequisites
+Your IoT Core Thing's [Policy](https://docs.aws.amazon.com/iot/latest/developerguide/iot-policies.html) must provide privileges for this sample to connect as well as subscribe, publish,
```

Comment:
"must provide privileges for this sample to connect as well as subscribe, publish, and receive as necessary to perform all of the data plane operations. Below is a sample policy that can be used on your IoT Core Thing that will allow this sample to run as intended."
->
"must provide privileges for this sample to connect, subscribe, publish, and receive in order to perform its data plane operations. Below is a sample policy that can be used on your IoT Core Thing that will allow this sample to run as intended."